### PR TITLE
feat: Remove rarities with oracle FF

### LINKED ISSUE #522: Cleanup rarities with oracle feature flag when the feature has been fully deployed on all envs

### PR DESCRIPTION
**Pull request by fzavalia**
Closes #522

### CODE REVIEW

File: src/Rarity/Rarity.router.spec.ts [hunk 1]
```diff
@@ -76,128 +73,81 @@ beforeEach(() => {
...
-        mockGetRarityFromBlockchain.mockResolvedValueOnce({
-          ...r,
-          price: priceMana,
-        })
-      }
+  describe('when fetching a rarity from the blockchain fails', () => {
+    it('should fail with a could not fetch from blockchain error', async () => {
+      mockGetRarityFromBlockchain.mockImplementation(() =>
+        Promise.reject(new Error('Atahualpa Yupanqui'))
+      )
```
Comment (LautaroPetaccio) [inline src/Rarity/Rarity.router.spec.ts:102]:
```suggestion
      mockGetRarityFromBlockchain.mockRejectedValueOnce(new Error('Atahualpa Yupanqui')
      )
```

File: src/Rarity/utils.ts [hunk 2]
```diff
@@ -1,13 +1,8 @@
-import { env } from 'decentraland-commons'
 import { ContractName, getContract } from 'decentraland-transactions'
 import { ethers } from 'ethers'
 import { getMappedChainIdForCurrentChainName } from '../ethereum/utils'
 import { getRpcUrl } from '../utils/eth'
 
-export function isUsingRaritiesWithOracle(): boolean {
-  return env.get<string | undefined>('FF_RARITIES_WITH_ORACLE') === '1'
```
Comment (LautaroPetaccio) [inline src/Rarity/utils.ts:8]: Would you mind removing the `FF_RARITIES_WITH_ORACLE` env variables from the CI and from the example?

**Review by LautaroPetaccio — CHANGES_REQUESTED** [review]

**Comment by fzavalia** [comment]
@LautaroPetaccio Reverted the suggestion because that function is being called multiple times and using "Once" will just mock it 1 time.

It is cleared on the root before each so it will not cause any problems.

Changed it to mockRejectedValue instead of the previous mockImplementation still.

File: src/Rarity/Rarity.router.spec.ts [hunk 3]
```diff
@@ -76,128 +73,81 @@ beforeEach(() => {
...
-    beforeEach(() => {
-      mockIsUsingRaritiesWithOracle.mockReturnValueOnce(false)
-    })
+  it('should return a list of rarities with the price converted to MANA from USD', async () => {
+    for (const r of rarities) {
+      mockGetRarityFromBlockchain.mockResolvedValueOnce({
+        ...r,
+        price: priceMana,
+      })
+    }
```
Comment (LautaroPetaccio) [inline src/Rarity/Rarity.router.spec.ts:82]: As this is building the context, would you mind putting it into a `beforeEach`?

File: src/Rarity/Rarity.router.spec.ts [hunk 4]
```diff
@@ -76,128 +73,81 @@ beforeEach(() => {
...
-        mockGetRarityFromBlockchain.mockResolvedValueOnce({
-          ...r,
-          price: priceMana,
-        })
-      }
+  describe('when fetching a rarity from the blockchain fails', () => {
+    it('should fail with a could not fetch from blockchain error', async () => {
+      mockGetRarityFromBlockchain.mockRejectedValue(
+        new Error('Atahualpa Yupanqui')
+      )
```
Comment (LautaroPetaccio) [inline src/Rarity/Rarity.router.spec.ts:102]: Same as before, would you mind placing the mock in a `beforeEach`?

File: src/Rarity/Rarity.router.spec.ts [hunk 5]
```diff
@@ -76,128 +73,81 @@ beforeEach(() => {
...
 })
 
 describe('when fetching a single rarity by name', () => {
-  describe('when rarities with oracle feature flag is disabled', () => {
-    beforeEach(() => {
-      mockIsUsingRaritiesWithOracle.mockReturnValueOnce(false)
+  it('should return the rarity with its price converted from USD to MANA', async () => {
+    mockGetRarityFromBlockchain.mockResolvedValueOnce({
+      ...rarities[0],
+      price: priceMana,
```
Comment (LautaroPetaccio) [inline src/Rarity/Rarity.router.spec.ts:121]: Same as before.

**Review by LautaroPetaccio — CHANGES_REQUESTED** [review]

**Review by LautaroPetaccio — APPROVED** [review]

**Pull request merged**